app/predacons_cli/src/rag.py

```
@@ -0,0 +1,174 @@
+from langchain_community.document_loaders import DirectoryLoader
+from langchain.text_splitter import RecursiveCharacterTextSplitter
+from langchain.schema import Document
+from langchain_community.vectorstores import Chroma
+from predacons import PredaconsEmbedding
+import requests
```

Ruff

`requests` imported but unused

Remove unused import: `requests`

(F401)